xformers/ops/fmha/triton_splitk.py

```diff
@@ -637,7 +637,7 @@ def dequantize(
             x_[:, :, None, :] >> offsets
         )  # (BLOCK_N, D // PACKED_PER_VAL, PACKED_PER_VAL)
 
-        quant_offset = tl.reshape(
+        quant_offset = tl.view(
```

Comment:
I think tl.view, which we used to use here and which used to do a non obvious thing, still officially has unspecified behaviour in triton. tl.reshape actually claims to return the right thing, which is nice 😊. Are you changing this here because you observe a speedup, or because you expect one, or is this something else e.g. a fix?

Reply:
I saw an error complaining no `tl.reshape`, and suggesting use `tl.view`. Let me double check.